src/test/java/nl/beehivejava/scriptj/parser/JsonPluginInformationParserBuilderTest.java

```diff
@@ -0,0 +1,129 @@
+package nl.beehivejava.scriptj.parser;
+
+import com.eclipsesource.json.Json;
+import com.eclipsesource.json.JsonObject;
+import nl.beehivejava.scriptj.util.builder.ObjectBuilders;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+
+/**
+ * @author Lesley
+ */
+public final class JsonPluginInformationParserBuilderTest extends PluginInformationParserTest {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') Add some tests to this class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS2187)